packages/skpm/src/commands/publish.js

```diff
@@ -124,16 +124,21 @@ export default asyncCommand({
 
     let tag = skpmConfig.version
 
-    const spinner = ora({
-      text: `Checking if \`${repo}\` is accessible`,
```

Comment:
I think you need to print at the beginning now

Reply:
This is done in on line 145, right?

https://github.com/idriesalbender/skpm/blob/29a58b3b69c0bae16ed721924b71e66a263feae0/packages/skpm/src/commands/publish.js#L145

Reply:
Ah you're right